engine/src/org/pentaho/di/trans/steps/fileinput/text/TextFileInputMetaInjection.java

```diff
@@ -0,0 +1,717 @@
+/*! ******************************************************************************
+ *
+ * Pentaho Data Integration
+ *
+ * Copyright (C) 2002-2013 by Pentaho : http://www.pentaho.com
+ *
+ *******************************************************************************
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
+
+package org.pentaho.di.trans.steps.fileinput.text;
+
+import static org.pentaho.di.trans.step.StepInjectionUtil.getEntry;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+import org.pentaho.di.core.Const;
+import org.pentaho.di.core.exception.KettleException;
+import org.pentaho.di.core.row.ValueMeta;
+import org.pentaho.di.core.row.ValueMetaInterface;
+import org.pentaho.di.trans.step.StepInjectionMetaEntry;
+import org.pentaho.di.trans.step.StepMetaInjectionEntryInterface;
+import org.pentaho.di.trans.step.StepMetaInjectionInterface;
+
+/**
+ * This takes care of the external metadata injection into the TextFileInputMeta class
+ *
+ * @author Matt
+ */
+public class TextFileInputMetaInjection implements StepMetaInjectionInterface {
+
+  enum Entry implements StepMetaInjectionEntryInterface {
```

Comment:
Yes, it happen after Eclipse's formatting. Looks like Eclipse's formatting settings don't 100% correrspond with checkstyle settings.